ew_agents/coordinator_integration.py

```
@@ -21,11 +26,1588 @@
+            if "risk_assessment" in nlp:
+                nlp_risk = nlp["risk_assessment"].get("overall_risk_score", 0.0)
+                risk_score += nlp_risk * 2.0
+        
+        return min(risk_score, 10.0)  # Cap at 10.0
+    
+    def _assess_violence_potential_detailed(self, agent_results: Dict[str, Any]) -> str:
+        """Detailed violence potential assessment"""
+        risk_score = self._calculate_comprehensive_risk_score(agent_results)
+        
+        # Check for specific violence indicators
+        violence_keywords = ["violence", "attack", "kill", "destroy", "eliminate"]
```

Ruff (0.12.2)

Local variable `violence_keywords` is assigned to but never used

Remove assignment to unused variable `violence_keywords`

(F841)